x-pack/test/functional/apps/ml/anomaly_detection/single_metric_job_without_datafeed_start.ts

```diff
@@ -0,0 +1,154 @@
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+
+  const jobId = `fq_single_1_${Date.now()}`;
+  const aggAndFieldIdentifier = 'Mean(responsetime)';
+  const bucketSpan = '30m';
+
+  function getExpectedRow(expectedJobId: string) {
+    return {
+      id: expectedJobId,
+      description: '',
+      jobGroups: [],
+      recordCount: '0',
+      memoryStatus: 'ok',
+      jobState: 'closed',
+      datafeedState: 'stopped',
+      latestTimestamp: '',
+    };
+  }
+
+  function getExpectedCounts(expectedJobId: string) {
+    return {
+      job_id: expectedJobId,
+      processed_record_count: '0',
+      processed_field_count: '0',
+      input_bytes: '0.0 B',
+      input_field_count: '0',
+      invalid_date_count: '0',
+      missing_field_count: '0',
+      out_of_order_timestamp_count: '0',
+      empty_bucket_count: '0',
+      sparse_bucket_count: '0',
+      bucket_count: '0',
+    };
+  }
+
+  function getExpectedModelSizeStats(expectedJobId: string) {
+    return {
+      job_id: expectedJobId,
+      result_type: 'model_size_stats',
+      total_by_field_count: '0',
+      total_over_field_count: '0',
+      total_partition_field_count: '0',
+      bucket_allocation_failures_count: '0',
+      memory_status: 'ok',
+    };
+  }
+
+  const calendarId = `wizard-test-calendar_${Date.now()}`;
+
+  describe('single metric without datafeed start', function () {
+    this.tags(['mlqa']);
+    before(async () => {
+      await esArchiver.loadIfNeeded('ml/farequote');
+      await ml.testResources.createIndexPatternIfNeeded('ft_farequote', '@timestamp');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+
+      await ml.api.createCalendar(calendarId);
+      await ml.securityUI.loginAsMlPowerUser();
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
+
+    it('job creation loads the single metric wizard for the source data', async () => {
+      await ml.testExecution.logTestStep('job creation loads the job management page');
+      await ml.navigation.navigateToMl();
+      await ml.navigation.navigateToJobManagement();
+
+      await ml.testExecution.logTestStep('job creation loads the new job source selection page');
+      await ml.jobManagement.navigateToNewJobSourceSelection();
+
+      await ml.testExecution.logTestStep('job creation loads the job type selection page');
+      await ml.jobSourceSelection.selectSourceForAnomalyDetectionJob('ft_farequote');
+
+      await ml.testExecution.logTestStep('job creation loads the single metric job wizard page');
+      await ml.jobTypeSelection.selectSingleMetricJob();
+    });
+
+    it('job creation navigates through the single metric wizard and sets all needed fields', async () => {
+      await ml.testExecution.logTestStep('job creation displays the time range step');
+      await ml.jobWizardCommon.assertTimeRangeSectionExists();
+
+      await ml.testExecution.logTestStep('job creation sets the timerange');
```

Comment:
super duper nit: I think it's `time range` here

Reply:
fixed for all similar tests in https://github.com/elastic/kibana/pull/77484/commits/45319031aad5dc64959f1c210953658cbc0a2bbb